packages/react-icons/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "outDir": "lib",
-    "target": "ES6",
+    "target": "esnext",
     "module": "esnext",
     "lib": ["es6", "dom"],
```

Comment:
Reduce some size. Because we already use `"module": "esnext"` before, so no breaking changes.